dotnu/mod.nu

```diff
@@ -13,5 +13,3 @@ export use commands.nu [
     "module-commands-code-to-record"
     "set-x"
 ]
-
-use ('..' | path join tests nupm utils dirs.nu) find-root
```

Comment:
I tried a conditional alternative (since tests aren't added on `nupm install`) but there doesn't seem to be a way to invoke `use` conditionally, however `find-root` wasn't actually used in the `dotnu` module.